src/Simulation/QSharpFoundation/Bitwise/Bitwise.qs

```diff
@@ -83,7 +83,17 @@ namespace Microsoft.Quantum.Bitwise {
     /// let x = Parity(a); // x : Int = 1.
     /// ```
     function Parity (a : Int) : Int {
-        body intrinsic;
+        mutable tmpParam = a;
```

Comment:
It looks like the algorithm here is to clear bits one at a time, counting how many iterations are needed to do so? In that case, I'm not sure `tmpParam` communicates that algorithm? In particular, `tmpParam` isn't a parameter, such that that's a bit confusing. Perhaps `remaining`?

Reply:
Yes, the algorithm is to clear bits one at a time. But instead of counting the iterations it flips the parity bit (the least significant bit in the `result` variable).
Renaming `tmpParam` to `remainingOnes`.

Reply:
Not applicable any more.